README.md

```diff
@@ -1,71 +1,188 @@
+```bash
+npm install
+```
 
-### `npm run build` fails to minify
+### Run Tests
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
-# math-magician
+To run tests, use the following commands:
+
+- To track CSS linter errors, run:
+```bash
+npx stylelint "**/*.{css,scss}"
+```
+- To track JavaScript linter errors, run:
+```bash
+npx eslint "**/*.{js,jsx}"
+```
+
+### Build
+
+To build the project, run the following command:
+
+```bash
+npm run build
+```
+
+### Webpack Development Server
+
+To run the webpack development server, execute the following command:
+
+```bash
+npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+- GitHub: [@DjibrilMugisho](https://github.com/DjibrilM?tab=overview&from=2023-04-01&to=2023-04-26)
+- Twitter: [@djibrilM](https://twitter.com/DjibrilMugisho)
+- LinkedIn: [DjibrilMugisho](https://www.linkedin.com/in/djibril-mugisho-509042241/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔮 Future Features <a name="future-features"></a>
+
+- [ ] 📱 **Add Animation**: Enhance the user interface with smooth animations and transitions to make the user experience more engaging and interactive.
+
+- [ ] 📝 **Add Model**: Implement a mathematical model or algorithm to perform advanced calculations, such as trigonometric functions or logarithms, providing users with a wider range of mathematical operations.
+- [ ] **styling**
+- [ ] **calculator**
+- [ ] **testing**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions to Math Magicians are welcome! If you would like to contribute, please follow these guidelines:
+
+1. Fork the repository and create a new branch for your contribution.
+2. Make your changes and ensure that the code is properly formatted.
+3. Write clear and concise commit messages.
+4. Push your branch to your forked repository.
+5. Submit a pull request detailing the changes you've made.
+
+Feel free to check the [issues page](https://github.com/kit0-0/math-magicians/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SHOW YOUR SUPPORT -->
+
+## ⭐️ Show Your Support <a name="show-your-support"></a>
+
+If you like this project, give it a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgments"></a>
+
+I would like to thank Microverse for providing guidance and support during the development of this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
```

Comment:
Great job @DjibrilM 🤝

- [ ] Kindly fix the broken MIT license link to make your readme file more professional.